packages/fuel-indexer-database/postgres/migrations/20231012190730_indexer_status.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS index_status (
+   id bigserial primary key,
```

Comment:
@lostman Should this not be a foreign key to `index_registry`?

Reply:
@lostman No this is fine as is :)

Reply:
@lostman Also, I see you commented under this particular comment about the foreign key 😆 . So before I resolve this thread, was there a reason `index_status` doesn't use a foreign key pointing to `index_registry` ?